services/firebase/src/firebase.ts

```diff
@@ -1,6 +1,9 @@
 import * as admin from 'firebase-admin'
+const serviceAccount = require('../serviceAccountKey.json');
```

Comment:
don't touch this file, revert it as it was.
to define a serviceAccount you need to edit .env file in services/firebase:

```
GOOGLE_APPLICATION_CREDENTIALS="/full/path/to/serviceAccountKey.json"
```